main.tf

```diff
@@ -1,7 +1,3 @@
-locals {
```

Comment:
this can be fixed with:

```hcl
locals {
  workspace_id = var.create && var.create_workspace ? aws_prometheus_workspace.this[0].id : var.workspace_id
}

Reply:
Fixed in new commit.

Thanks a ton !